QgisModelBaker/gui/import_data.py

```diff
@@ -291,7 +336,9 @@ def enable(self):
         self.buttonBox.setEnabled(True)
 
     def type_changed(self):
+        self.remove_import_without_validate_button()
         self.progress_bar.hide()
+        self.txtStdout.clear()
```

Comment:
Is that related to this change? Could this be done in a separate pull request?

Reply:
yes, the change was made in the PR #334